hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/store/impl/TestSQLFederationStateStore.java

```diff
@@ -592,4 +588,14 @@ public void testRemoveStoredToken() throws IOException, YarnException {
   public void testGetTokenByRouterStoreToken() throws IOException, YarnException {
     super.testGetTokenByRouterStoreToken();
   }
+
+  @Override
+  protected void checkRouterMasterKey(DelegationKey delegationKey,
+      RouterMasterKey routerMasterKey) throws YarnException, IOException {
```

Comment:
Comment on why we do nothing

Reply:
Thank you for your suggestion! I'll add a comment explaining what the method does.